src/Appium.Net/Appium/ScreenOrientationExtensions.cs

```diff
@@ -27,7 +27,7 @@ public static class ScreenOrientationExtensions
         /// <param name="orientation"></param>
         /// <returns></returns>
         public static string JSONWireProtocolString(this ScreenOrientation orientation) =>
-            orientation.ToString().ToUpper();
+            orientation.ToString().ToUpperInvariant();
```

Comment:
Related tests passed?

Reply:
I tested the enum to string conversion on my local with `ToUpperInvariant`, while my env didn't cause the https://github.com/appium/dotnet-client/pull/920, so it tested the output didn't change with `ToUpper` though